packages/providers/mgt-teamsfx-provider/src/TeamsFxProvider.ts

```diff
@@ -68,11 +76,19 @@ export class TeamsFxProvider extends IProvider {
    */
   private _accessToken: string = '';
 
-  constructor(teamsfx: TeamsFx, scopes: string | string[], baseURL: GraphEndpoint = MICROSOFT_GRAPH_DEFAULT_ENDPOINT) {
+  constructor(teamsfx: TeamsFx, scopes: string | string[], baseURL?: GraphEndpoint);
+  constructor(teamsUserCredential: TeamsUserCredential, scopes: string | string[], baseURL?: GraphEndpoint);
+  constructor(authConfig: TeamsFx | TeamsUserCredential, scopes: string | string[], baseURL?: GraphEndpoint) {
     super();
```

Comment:
@gavinbarron I made a conflict resolution here but I'm unsure of the changes now after seeing this https://github.com/microsoftgraph/microsoft-graph-toolkit/pull/1983. Can you direct on the way forward for this?

Reply:
@musale I think we should get this merged and look to have that PR updated if necessary

Reply:
Hi, @musale and @gavinbarron , I have created another PR to update the TeamsFxProvider.ts for v3, and when this PR has been merged, I will resolve conflict for that PR:

https://github.com/microsoftgraph/microsoft-graph-toolkit/pull/1983